MapboxNavigation/Resources/tr.lproj/Localizable.strings

```diff
@@ -0,0 +1,129 @@
+
+/* Message when search returned zero results in CarPlay */
+"CARPLAY_SEARCH_NO_RESULTS" = "Sonuç bulunamadı";
+
+/* Alert title that shows when feedback has been submitted */
+"CARPLAY_SUBMITTED_FEEDBACK" = "Gönderildi";
+
+/* Title for unmute button */
+"CARPLAY_UNMUTE" = "Sesi aç";
+
+/* Dismiss button title on the steps view */
+"DISMISS_STEPS_TITLE" = "Kapat";
+
+/* Title used for arrival */
+"END_OF_ROUTE_ARRIVED" = "Hedefe vardınız";
+
+/* Comment Placeholder Text */
+"END_OF_ROUTE_TITLE" = "How can we improve?";
+
+/* Error message when the SDK is unable to read a spoken instruction. */
+"FAILED_INSTRUCTION" = "Unable to read instruction aloud.";
+
+/* Indicates a faster route was found */
+"FASTER_ROUTE_FOUND" = "Daha hızlı bir rota bulundu";
+
+/* Feedback type for Bad Route */
+"FEEDBACK_BAD_ROUTE" = "Kötü\nRota";
+
+/* Feedback type for Confusing Instruction */
+"FEEDBACK_CONFUSING_INSTRUCTION" = "Kafa karıştıran talimat";
+
+/* Feedback type for Other Map Issue Issue */
+"FEEDBACK_GENERAL_ISSUE" = "Diğer harita sorunu";
+
+/* Feedback type for a maneuver that is Not Allowed */
+"FEEDBACK_NOT_ALLOWED" = "Erişime kapalı";
+
+/* Feedback type for Report Traffic */
+"FEEDBACK_REPORT_TRAFFIC" = "Trafik yoğunluğu";
+
+/* Feedback type for Road Closed */
+"FEEDBACK_ROAD_CLOSURE" = "Yol kapalı";
+
+/* Message confirming that the user has successfully sent feedback */
+"FEEDBACK_THANK_YOU" = "Geri bildiriminiz için teşekkür ederiz.";
+
+/* Title of view controller for sending feedback */
+"FEEDBACK_TITLE" = "Sorun bildir.";
+
+/* Label indicating the device volume is too low to hear spoken instructions and needs to be manually increased */
+"INAUDIBLE_INSTRUCTIONS_CTA" = "Sesli talimatları duyabilmek için sesi yükseltin.";
+
+/* Format for displaying the first two major ways */
+"LEG_MAJOR_WAYS_FORMAT" = "%1$@ ve %2$@";
+
+/* Format string for a short distance or time less than a minimum threshold; 1 = duration remaining */
+"LESS_THAN" = "<%@";
+
+/* Accessibility value of label indicating the absence of a rating */
+"NO_RATING" = "Derecelendirme yok";
+
+/* Rating Reset To Zero Accessability Hint */
+"RATING_ACCESSIBILITY_RESET" = "Derecelendirmeyi sıfırlamak için dokunun.";
+
+/* Format for accessibility label of button for setting a rating; 1 = number of stars */
+"RATING_ACCESSIBILITY_SET" = "%ld-yıldız ile derecelendir.";
+
+/* Format for accessibility value of label indicating the existing rating; 1 = number of stars */
+"RATING_STARS_FORMAT" = "%ld yıldız.";
+
+/* Indicates that rerouting is in progress */
+"REROUTING" = "Yeniden yönlendiriliyor…";
+
+/* Button title for resume tracking */
+"RESUME" = "Devam et";
+
+/* Label above the speed limit in an MUTCD-style speed limit sign. Keep as short as possible. */
+"SPEED_LIMIT_LEGEND" = "Max";
+
+/* The text of a banner that appears during turn-by-turn navigation when route simulation is enabled. */
+"USER_IN_SIMULATION_MODE" = "Simulating Navigation at %@×";
```

Comment:
Not sure if this is expected, but I still see some English in Turkish translation.

Reply:
This file is at least 80% translated, so the repository is configured to pull down the whole thing:

https://github.com/mapbox/mapbox-navigation-ios/blob/f2f78ce9ebd7d411cbb907e479b5bc5327b3bd85/.tx/config#L4